lib/rubycritic/analysers_runner.rb

```diff
@@ -25,13 +24,15 @@ def initialize(paths)
 
     def run
       ANALYSERS.each { |analyser| analyser.new(analysed_modules).run }
+      puts "The (limited) average cost is: #{analysed_modules.average_cost}"
```

Comment:
Maybe we should merge these PR as is and refactor the existing reports with these pieces elsewhere so we don't stall the existing PRs too much :)